src/com/nicoles_change_maker/ChangeMaker.java

```diff
@@ -0,0 +1,61 @@
+package com.nicoles_change_maker;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Iterator;
+/**
+ * Created by bears8yourface on 1/5/16.
+ */
+public class ChangeMaker {
+    double remainingDollars;
+
+    public HashMap makeChange(double dollarAmount) {
+        remainingDollars = dollarAmount;
+
+        if (checkDollarAmountIsValid(dollarAmount)) {
+            return evaluateEachCoinAmount();
+        }
+        throw new IllegalArgumentException();
+    }
+
+    private HashMap evaluateEachCoinAmount() {
```

Comment:
"Evaluate" doesn't seem like quite the right verb here. What does it actually mean to "evaluate a coin amount?"